src/DnsClient/LookupClient.cs

```diff
@@ -482,6 +482,22 @@ public IDnsQueryResponse Query(DnsQuestion question, DnsQueryAndServerOptions qu
 
             var settings = GetSettings(queryOptions);
             return QueryInternal(question, settings, settings.ShuffleNameServers());
+        }
+
+        /// <inheritdoc/>
+        public IDnsQueryResponse QueryCache(string query, QueryType queryType, QueryClass queryClass = QueryClass.IN)
```

Comment:
The 2 calls look good I guess.
I wasn't actually thinking of returning the real result and rather have an IsCached method.
But I guess that's fine, too.

Could you add the 2 methods to the ILookupClient interface?